core/src/main/scala/com/nvidia/spark/rapids/tool/planparser/SQLPlanParser.scala

```diff
@@ -76,7 +76,12 @@ object SQLPlanParser extends Logging {
 
   val windowFunctionPattern = """(\w+)\(""".r
 
-  val ignoreExpressions = Array("any", "cast", "decimal", "decimaltype", "every", "some", "list")
+  val ignoreExpressions = Array("any", "cast", "decimal", "decimaltype", "every", "some",
+    "list",
+    // current_database does not cause any CPU fallbacks
+    "current_database",
+    // ArrayBuffer is a Scala and may appear in some of the JavaRDDs/UDAFs)
```

Comment:
typo?  s/is a Scala/is a Scala function/

Reply:
fixed